tool/flutter_ci_script_stable.sh

```diff
@@ -22,6 +22,8 @@ declare -ar PROJECT_NAMES=(
     "experimental/federated_plugin/federated_plugin"
     "experimental/web_dashboard"
     "experimental/linting_tool"
+    # TODO(DomesticMouse): include when stable increments
+    # "experimental/varfont_shader_puzzle"
```

Comment:
What is it that precludes this sample from running on Flutter stable? If there is a feature in Flutter 3.7.0-beta, please mark up the `pubspec.yaml` as having a minimum supported Flutter version.

Reply:
`FragmentShader` and the `toImageSync()` method for `RepaintRenderBoundary` are only available in master and beta at the moment, so I've commented this project out of stable for now.

I've marked the current beta release as minimum in `pubspec.yaml` since the `FragmentShader` API changed somewhat recently, and my code will break under the older API, but I can't tell exactly which beta release that API change occurred in.